### PR TITLE
Update face-detection-adas-0001.md

### PR DESCRIPTION
**Pull request by Aya-ZIbra**
The outputs are sorted by confidence decrease.

### CODE REVIEW

**Comment by vladimir-dudnik** [comment]
@Aya-ZIbra please resolve conflicts

File: models/intel/face-detection-adas-0001/description/face-detection-adas-0001.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ curve. Numbers are on
 ## Outputs
 
 1. The net outputs blob with shape: [1, 1, N, 7], where N is the number of detected
-   bounding boxes. For each detection, the description has the format:
+   bounding boxes. The results are sorted by confidence decrease. For each detection, the description has the format:
```
Comment (snosov1) [inline models/intel/face-detection-adas-0001/description/face-detection-adas-0001.md:47]:
```suggestion
   bounding boxes. The results are sorted by confidence in decreasing order. For each detection, the description has the format:
```

**Comment by snosov1** [comment]
For some reason, it shows merge conflicts. Please, resolve. 

As this PR contains 7 commits for a single line change - maybe, it would be easier to simply create a new one.

**Comment by IRDonch** [comment]
> For some reason, it shows merge conflicts. Please, resolve.
> 
> As this PR contains 7 commits for a single line change - maybe, it would be easier to simply create a new one.

FYI, both the merge conflicts and the extra commits are there because this PR was originally targeted to `master`, and I switched it to `develop`.

@Aya-ZIbra Please rebase your branch on top of the `develop` branch.

**Comment by Aya-ZIbra** [comment]
@IRDonch Do I still have to rebase my branch for the changes to be reflected on the dev branch, please?

**Comment by IRDonch** [comment]
@Aya-ZIbra Yes, you need to rebase on top of the `develop` branch.

**Comment by Aya-ZIbra** [comment]
@IRDonch I did rebase on top of  `develop` branch.  I noticed some edits from another developer on the head were dismissed, though. How can I resolve this, please?

**Comment by IRDonch** [comment]
@Aya-ZIbra I see two problems on your branch right now:

1. It contains commit 8a830538, which is unrelated and shouldn't be there. The easiest way to get rid of it is probably to run `git rebase 8a830538 --onto origin/develop`.

2. You have committed conflict markers. You need to instead combine your change (the block after `=======`) with the upstream code (the block after `<<<<<<<`) and commit that.

**Comment by Aya-ZIbra** [comment]
@IRDonch Can you please check now? Looks good to me now. Thanks

**Comment by IRDonch** [comment]
Jenkins: test this please

**Pull request merged**